scriptlets/install_checkbox_debs

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Installs Checkbox snap (runtime + frontend) on the DUT. Also installs
+# Checkbox on the agent from source matching the version on the DUT
+
+if [[ "$#" != "1" ]]; then
+  echo "Usage: $(basename ${BASH_SOURCE[0]}) risk"
+  exit 1
+fi
+
+echo "Installing Checkbox snaps"
+
+RISK=$1
+
+wait_for_packages_complete
+_run sudo add-apt-repository ppa:checkbox-dev/$RISK
+_run install_packages checkbox-ng python3-checkbox-ng checkbox-provider-base checkbox-provider-resource checkbox-provider-sru fswebcam obexftp wmctrl iperf mesa-utils vim pastebinit fwts xorg-dev gir1.2-clutter-1.0
+wait_for_packages_complete
+
+export CHECKBOX_VERSION=$(_run checkbox-cli --version)
+[ -z "$CHECKBOX_VERSION" ] && echo "Error: Unable to retrieve Checkbox version from device" && exit 1
+
+check_for_checkbox_service
+
+echo "Installing checkbox $CHECKBOX_VERSION on the agent container from source"
+install_checkbox_agent_source $CHECKBOX_VERSION
```

Comment:
I am wondering if it would make sense to incorporate these lines into `install_checkbox_agent_source`. They are almost identical for both the snap and deb version, except for the fact that the FRONTEND_NAME is fixed (not an input) for the deb version.

If we were to do that, we wouldn't need to duplicate the code here. We would just call `install_checkbox_agent_source`, just making sure that `FRONTEND_NAME` is set in both cases.

One argument I can think of _against_ this suggestion is if there are cases when we would need to use `install_checkbox_agent_source` with an arbitrary version, instead of one we retrieve from the device.

Reply:
my argument for why this should call the other script and not the other way around is that this script knows how to fetch the checkbox version while the other doesnt